spec/compiler/formatter/formatter_spec.cr

```diff
@@ -581,6 +581,8 @@ describe Crystal::Formatter do
   assert_format "lib Foo\nend"
   assert_format "lib Foo\ntype  Foo  =   Bar\nend", "lib Foo\n  type Foo = Bar\nend"
   assert_format "lib Foo\nfun foo\nend", "lib Foo\n  fun foo\nend"
+  assert_format "lib Foo\nfun Bar\nend", "lib Foo\n  fun Bar\nend"
+  assert_format "lib Foo\nfun bar = Bar\nend", "lib Foo\n  fun bar = Bar\nend"
```

Comment:
What about `lib Foo\nfun Foo = Bar\nend`?

We should probably spec calling functions with `CONST` names too to make sure that doesn't get broken.

Reply:
about calling, do you mean add a formatter spec for the lib-uppercase-fun calling?

Reply:
@bew yes

Reply:
done!